webpack/assets/javascripts/react_app/redux/API/APIActions.js

```diff
@@ -0,0 +1,22 @@
+import { API_OPERATIONS } from './APIConstants';
+
+export const get = ({
+  key,
+  url,
+  params = {},
+  payload = {},
+  actionTypes = {},
+}) => ({
+  type: API_OPERATIONS.GET,
+  key,
+  url,
+  payload,
+  params,
+  actionTypes,
+});
+
+// here all of the other API operations will be added.
+
+export default {
+  get,
+};
```

Comment:
We don't have to mix named with default imports.
It can be removed and consumed as:
```js
import * as ApiActions from './ApiActions`;
```

And in the `API/index.js` it can be:
```js
export * as ApiActions from './APIActions`;
```